rotkehlchen/tests/db/test_db.py

```diff
@@ -1352,3 +1353,17 @@ def test_values_are_present_in_db(database, enum_class, table_name):
     for enum_class_entry in enum_class:
         r = cursor.execute(query, (enum_class_entry.value,))
         assert r.fetchone() == (1,)
+
+
+@pytest.mark.parametrize('enum_class, table_name', [(AssetType, 'asset_types')])
+def test_values_are_present_in_global_db(globaldb, enum_class, table_name):
+    """
+    Check that all enum classes have the same number of possible values
+    in the class definition as in the database
+    """
+    cursor = globaldb._conn.cursor()
+    query = 'SELECT COUNT(*) FROM {} WHERE seq=?'.format(table_name)
+
+    for enum_class_entry in enum_class:
+        r = cursor.execute(query, (enum_class_entry.value,))
+        assert r.fetchone() == (1,)
```

Comment:
Here you could have found the problem faster if you had an assertion error message

```suggestion
        assert r.fetchone() == (1,), f'Did not find {table_name} entry for value {enum_class_entry.value}'
```